### DISCUSSION TITLE
Allow mob griefing only by residents?

### DISCUSSION THREAD

**Discussion by vJoeyz** [TownyAdvanced/Towny#5592] [Q&A]
Is there a way to enable mob griefing on claim plots only by that town's residents?
For example, an outsider that lures a creeper on a claimed plot won't cause blocks to explode, but when a resident of that town does the same it will.

**Comment by LlmDl** [comment]
That sounds fairly complicated when the player could just have explosions turned off, or they could build a wall.

**Reply by vJoeyz** [reply]
I want to keep the vanilla behavior and consequences to one's actions (eg making a creeper explode), without enabling outsiders to grief town plots.

**Reply by LlmDl** [reply]
I'd recommend filing a suggestion ticket on the issue tracker if you want a feature to be considered.
